uct/prior.c

```diff
@@ -113,13 +113,87 @@ print_dcnn_best_moves(struct tree_node *node, struct prior_map *map,
         fprintf(stderr, "]\n");	
 }
 
+static float*
+dcnn2_get_moves_data(struct board *b, enum stone color)
+{
+	assert(real_board_size(b) == 19);
+	int size = 19;
+	int dsize = 13 * size * size;
+	float *data = (float *)malloc(dsize * sizeof(float));
+	for (int i = 0; i < dsize; i++) {
+		data[i] = 0.0;
+	}
+	for (int j = 0; j < size; j++) {
+		for(int k = 0; k < size; k++) {
+			int p = size * j + k;
+			coord_t c = coord_xy(b, j+1, k+1);
+			struct group g = board_group_info(b, group_at(b, c));
+			enum stone bc = board_at(b, c);
+			int libs = g.libs - 1;
+			if (libs > 3) libs = 3;
+			if(bc == S_NONE) {
+				data[8*size*size + p] = 1.0;
+			} else if (bc == color) {
+				data[(0+libs)*size*size + p]=1.0;
+			} else if (bc == stone_other(color)) {
+				data[(4+libs)*size*size + p]=1.0;
+			}
+			if(c == b->last_move.coord) {
+				data[9*size*size + p] = 1.0;
+			} else if (c == b->last_move2.coord) {
+				data[10*size*size + p] = 1.0;
+			} else if (c == b->last_move3.coord) {
+				data[11*size*size + p] = 1.0;
+			} else if (c == b->last_move4.coord) {
+				data[12*size*size + p] = 1.0;
+			}
+		}
+	}
+	return data;
+}
+
+static float*
+dcnn_get_moves_data(struct board *b, enum stone color)
+{
+	assert(real_board_size(b) == 19);
+	int size = 19;
+	float *data = (float *)malloc(2 * size * size * sizeof(float));
+	
+	if (color == S_BLACK)
+		for (int j = 0; j < size; j++)
+			for (int k = 0; k < size; k++) {
+				coord_t c = coord_xy(b, j + 1, k + 1);
+				int offsetb = j * size + k;
+				int offsetw = size * size + size * j + k;
+				fprintf(stderr, "board_at %d : %d\n", c, board_at(b, c));
+				data[offsetb] = (board_at(b, c) == S_BLACK ? 1.0 : 0.0);
+				data[offsetw] = (board_at(b, c) == S_WHITE ? 1.0 : 0.0);
+			}
+	
+	if (color == S_WHITE) {
+		for (int j = 0; j < size; j++)
+			for (int k = 0; k < size; k++) {
+				coord_t c = coord_xy(b, j + 1, k + 1);
+				int offsetb = j * size + k;
+				int offsetw = size * size + size * j + k;
+				fprintf(stderr, "board_at %d : %d\n", c, board_at(b, c));
+				data[offsetb] = (board_at(b, c) == S_WHITE ? 1.0 : 0.0);
+				data[offsetw] = (board_at(b, c) == S_BLACK ? 1.0 : 0.0);
+			}
+	}
+
+	return data;
+}
+
```

Comment:
If we aren't using dcnn_get_moves_dta() anywhere, let's remove it completely. Do you agree?